kotlin/src/test/java/com/squareup/moshi/KotlinJsonAdapterTest.kt

```diff
@@ -25,9 +25,12 @@ import java.util.SimpleTimeZone
 import kotlin.annotation.AnnotationRetention.RUNTIME
 
 class KotlinJsonAdapterTest {
+  /** Returns a new instance of [Moshi] with [KotlinJsonAdapterFactory] added. */
+  val newMoshi: Moshi
```

Comment:
Although it's a `val` in bytecode this will be called as a getter method, so we can be sure that tests are not affecting each other.